build_helper.sh

```diff
@@ -165,7 +165,11 @@ function setup_fmt() {
 
   if [ ! -d "$FMT_DIR" ] ; then
     echo -e "${COLOR_GREEN}[ INFO ] Cloning fmt repo ${COLOR_OFF}"
+<<<<<<< HEAD
```

Comment:
Thanks for the PR. 
I have imported this PR for internal review. 
If you make significant changes let me know, I'll have to re-import it. 
This looks like a bad merge ?

Reply:
Thanks! 
I found there will a problem if just pull 1 lastest commit, because the deps commit in mvfst will a few commits behind. Then I change to 10 lastest commits, as we just pull at the first time, which will insure a bit.

Reply:
ah I see. thanks for catching it. 

Yeah, in general there is no guarantee here. 
May I ask how slow is the checkout for you ?

Reply:
1. It depends on the network, sometimes will be 40 kb/s, and better condition will be 150 kb/s.
2. I'm tested with the --depth=10 in almost 100kb/s network. It costed me 1min and without --depth=10 costed me 4mins. I think it depends on the git receiving objects.

![test_result](https://user-images.githubusercontent.com/10419854/82106885-6a064980-9756-11ea-806b-e8be29a8878d.PNG)